pages/linux/timedatectl.md

```diff
@@ -0,0 +1,19 @@
+# timedatectl
+
+> Control the system time and date
+
+- To check the current system clock time
+
+`timedatectl`
+
+- To set the local time of the system clock directly
+
+`timedatectl set-time "yyyy-MM-dd hh:mm:ss"`
```

Comment:
Yep, exactly, since the part in quotes is a placeholder for an actual timestamp (if I'm reading this correctly), then it should be wrapped in the placeholder/token syntax.